src/Arcus.Templates.ServiceBus.Queue/Arcus.Templates.ServiceBus.Queue.csproj

```diff
@@ -55,7 +55,7 @@
     <PackageReference Include="Serilog" Version="2.9.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Extensions.Hosting" Version="3.0.0" Condition="'$(ExcludeSerilog)' == 'false'" />
     <PackageReference Include="Serilog.Sinks.ApplicationInsights" Version="3.1.0" Condition="'$(ExcludeSerilog)' == 'false'" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.9.10" />
+    <PackageReference Include="Serilog.Sinks.Console" Version="3.1.1" Condition="'$(Configuration)' == 'Debug'"/>
```

Comment:
It's not a removal, apperently, the Container tools package was duplicated here.